tests/unit/h/models/subscriptions_test.py

```diff
@@ -9,5 +9,5 @@ def test___repr__(self):
 
         assert (
             repr(subscription)
-            == "<Subscription uri=http://example.com type=reply active=True>"
+            == "<Subscription uri=http://example.com type=Type.REPLY active=True>"
```

Comment:
Not sure about the need for this repr and this test but I will just make it pass in the new python version.